Allow pkg-config to find specifig version of python-embed.pc
--- configure.ac.orig	2022-05-23 20:57:52.000000000 +0300
+++ configure.ac	2022-05-23 20:59:41.000000000 +0300
@@ -100,7 +100,7 @@
         rm -f Python.framework
         ln -s $PYTHON_FRAMEWORK Python.framework ])
     PKG_CHECK_MODULES([python], [python-embed], [PYTHON_CONFIG_EXISTS=yes], [PYTHON_CONFIG_EXISTS=no])
-    PKG_CHECK_MODULES([python], [python3-embed], [PYTHON3_CONFIG_EXISTS=yes; AC_DEFINE(PY_IS_PYTHON3, [1], [Is Python version 3])], [PYTHON3_CONFIG_EXISTS=no])
+    PKG_CHECK_MODULES([python], [python-3.10-embed], [PYTHON3_CONFIG_EXISTS=yes; AC_DEFINE(PY_IS_PYTHON3, [1], [Is Python version 3.10])], [PYTHON3_CONFIG_EXISTS=no])
     if test "$PYTHON_CONFIG_EXISTS" = "yes" || test "$PYTHON3_CONFIG_EXISTS" = "yes"; then
         AM_CONDITIONAL([BUILD_PYTHON_API], [true])
         AC_DEFINE([HAVE_PYTHON], [1], [Python support])
